fastapi/utils.py

```diff
@@ -169,8 +167,7 @@ def get_value_or_default(
 
     Otherwise, the first item (a `DefaultPlaceholder`) will be returned.
     """
-    items = (first_item,) + extra_items
-    for item in items:
+    for item in (first_item,) + extra_items:
```

Comment:
It's a matter of taste of course, but I find the single line harder to read than the multi line code it replaces.

Reply:
Thanks @JarroVGIT for you review I appreciate it.
I thought this change will be more explicit, but I could agree with you, it might add some complexity to the code.
I'll roll back the changes using the previous multi line approach.